### PR TITLE
chapter 11 account activation

### PR DESCRIPTION
**Pull request by qtv06**

### CODE REVIEW

**Comment by qtv06** [comment]
ready

File: app/helpers/account_activations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (tranducquoc) [inline app/helpers/account_activations_helper.rb]: file k làm gì thì remove

File: app/views/user_mailer/account_activation.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1><%= t "title_header" %></h1>
+<p><%= t "mailer.hi_text" %></p>
+<p><%= t "mailer.text_welcome" %></p>
+<%= link_to t("mailer.title_activate"),
+ edit_account_activation_url(@user.activation_token, email: @user.email) %>
```
Comment (tranducquoc) [inline app/views/user_mailer/account_activation.html.erb]: indent

**Comment by tranducquoc** [comment]
commented

**Comment by qtv06** [comment]
ready

**Pull request merged**